src/main/java/Blitz.java

```diff
@@ -0,0 +1,55 @@
+import java.util.Scanner;
+
+import java.io.IOException;
+
+public class Blitz {
```

Comment:
Missing JavaDoc header comments for many of the public classes and constructors, hopefully you're in the process of writing them up! :blush: